meta/requirements.yml

```
@@ -6,3 +6,4 @@
 # See also cisagov/skeleton-ansible-role#153.
 - name: cyhy_core
   src: https://github.com/cisagov/ansible-role-cyhy-core
+  version: improvement/scanning-by-hostname
```

I know this will be removed, but right now the branch specified does not exist.

```suggestion
  version: testing/support-hostnames
```

---

Oops, good catch!  See https://github.com/cisagov/ansible-role-cyhy-commander/pull/88/commits/610a735e05e23ad211a442c4db95ac4b8b4b17b1.